idd/Energy+.idd.in

```diff
@@ -42366,7 +42372,6 @@ ZoneHVAC:EquipmentConnections,
        \object-list ZoneNames
   A2 , \field Zone Conditioning Equipment List Name
        \note Enter the name of a ZoneHVAC:EquipmentList object.
-       \required-field
```

Comment:
This should be restored now.

Reply:
@mjwitte @rraustad  Since you added a new input field for *System Inlet Air Node Name*, did you also want to increase the min-fields requirement by 1? *Latent Heat Recovery Effectiveness* used to be part of the list of minimum fields to define, and now it is not.

Reply:
Yes. That was an oversight. Thanks.

Reply:
@rraustad You can add that change to #6297

Reply:
Cool. You're welcome, I was just checking.